---
title: "Michigan Center for Single-Cell Genomic Data Analytics"
excerpt: "<img src='/images/PicturedRocks.jpg'> Data analysis for single-cell RNA seq data"
collection: portfolio
---

Jun Z. Li (Departments of Human Genetics and Computational Medicine and Bioinformatics) and I are the co-directors of the [Michigan Center for Single-Cell Genomic Data Analytics](http://midas.umich.edu/research/health/single-cell/). Our research team connects mathematicians and data scientists with biological researchers to develop, evaluate, and implement a variety of cutting-edge methodologies in sparse data analysis.

I have focused on genetic marker selection for single-cell RNA seq data. PicturedRocks is a tool for the analysis of single cell RNA-seq data. Currently, we implement two marker selection approaches: a 1-bit compressed sensing based sparse SVM algorithm and a mutual information-based Max Relevance min Redundancy algorithm. Marker selection is defined simply as feature selection (in machine learning).

We emphasize that this software and its attendant __rigorous__ analysis is very much a work in progress. We will be updating it as we determine better its empirical performance. Please contact us with your feedback and performance results!

- __Software:__ Anna C. Gilbert, Alexander Vargo, Umang Varma, [Pictured Rocks: Single Cell RNA-seq analysis tool](https://picturedrocks.github.io/), 2017.

